operator/src/main/java/org/bf2/operator/operands/AdminServer.java

```diff
@@ -120,6 +120,9 @@ public void createOrUpdate(ManagedKafka managedKafka) {
         if (openShiftClient != null) {
             Route currentRoute = cachedRoute(managedKafka);
             Route route = routeFrom(managedKafka, currentRoute);
+            if (ingressControllerManagerInstance.isResolvable()) {
+                ingressControllerManagerInstance.get().ensureBlueprintRouteMatching(route, "kafka-admin");
```

Comment:
The fact that the guard condition is a general `isResolvable` but we then only ensure a blueprint for the `kafka-admin` route, strikes me as a bit strange. 

API consistency is a longer term thing though. 

The binary decision that the instance is resolvable feels wrong to me, especially with enterprise deployments, I would expect that we would want to be able to make the instances themselves resolvable but keep the admin cluster only etc. This might all be covered as part of one of the networking epics so I'm just voicing the thought here so as not to loose it.

Reply:
isResolvable in this context is to allow for testing and minikube scenarios where the ingresscontrollermanager is not installed - it is not related to endpoint / dns resolving.